server/backend/cron/mealNameDownload.go

```diff
@@ -0,0 +1,192 @@
+package cron
+
+import (
+	"encoding/json"
+	"github.com/TUM-Dev/Campus-Backend/model/cafeteria_rating_models"
+	log "github.com/sirupsen/logrus"
+	"gorm.io/gorm"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+	"strings"
+	"time"
+)
+
+type CafeteriaName struct {
+	Name     string   `json:"enum_name"`
+	Location Location `json:"location"`
+}
+
+type CafeteriaWithID struct {
+	Name      string `json:"name"`
+	Cafeteria int32  `json:"cafeteria"`
+}
+
+type Location struct {
+	Longitude float32 `json:"longitude"`
+	Latitude  float32 `json:"latitude"`
+	Address   string  `json:"address"`
+}
+
+type Days struct {
+	Days []Date `json:"days"`
+}
+
+type Date struct {
+	Dates []Dish `json:"dishes"`
+}
+
+type Dish struct {
+	Name     string `json:"name"`
+	DishType string `json:"dish_type"`
+}
+
+//fileDownloadCron Downloads all files that are not marked as finished in the database.
+func (c *CronService) dishNameDownloadCron() error {
+
+	downloadCanteenNames(c)
+	downloadDailyDishes(c)
+
+	return nil
+}
+
+func downloadDailyDishes(c *CronService) {
+	var result []CafeteriaWithID
+	c.db.Model(&cafeteria_rating_models.Cafeteria{}).Select("name,id").Scan(&result)
+
+	for _, v := range result {
+
+		cafeteriaName := strings.Replace(strings.ToLower(v.Name), "_", "-", 10)
+		log.Println(cafeteriaName)
+		y, w := time.Now().UTC().ISOWeek()
+
+		req := "https://tum-dev.github.io/eat-api/" + cafeteriaName + "/" + strconv.Itoa(y) + "/" + strconv.Itoa(w) + ".json"
+		println(req)
+		var resp, err = http.Get(req)
+		if err != nil {
+			log.Fatalln(err)
+		}
+		body, err := ioutil.ReadAll(resp.Body)
+		if body[0] == '<' {
```

Comment:
Is this really the best way to check if nothing was returned? Please _actually_ check the status code.